src/components/symptom-shark/presentational/SymptomTreatmentHistograms/SymptomTreatmentHistograms.stories.tsx

```diff
@@ -1,22 +1,62 @@
-import React from "react";
-import { DateRangeCoordinator, Layout } from "../../../presentational";
-import SymptomTreatmentHistograms, { SymptomTreatmentHistogramsProps } from "./SymptomTreatmentHistograms";
-import { SymptomSharkVisualizationCoordinator } from "../../container";
+import React from 'react';
+import { Layout } from '../../../presentational';
+import SymptomTreatmentHistograms, { SymptomTreatmentHistogramsProps } from './SymptomTreatmentHistograms';
+import VisualizationCoordinator from '../../container/VisualizationCoordinator/VisualizationCoordinator';
 
-export default { title: "SymptomShark/Presentational/SymptomTreatmentHistograms", component: SymptomTreatmentHistograms, parameters: { layout: 'fullscreen' } };
-let render = (args: SymptomTreatmentHistogramsProps) => <Layout colorScheme="auto">
-    <DateRangeCoordinator intervalType={"Month"} variant="rounded">
-        <SymptomSharkVisualizationCoordinator previewState="default" showFilters>
-            <SymptomTreatmentHistograms {...args} />
-        </SymptomSharkVisualizationCoordinator>
-    </DateRangeCoordinator>
-</Layout>
+export default {
+    title: 'Symptom Shark/SymptomTreatmentHistograms',
```

Comment:
I think this should be `SymptomShark/Presentational/SymptomTreatmentHistograms` to be consistent with the other component groupings.